packages/sitecore-jss/src/feaas/themes.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  ComponentRendering,
+  HtmlElementRendering,
+  LayoutServiceData,
+  RouteData,
+  getFieldValue,
+} from '../layout';
+
+/**
+ * Pattern for library ids
+ * @example -library--foo
+ */
+const FEAAS_LIBRARY_ID_REGEX = /-library--([^\s]+)/;
+
+/**
+ * FEAAS CDN hostnames
+ */
+export const FEAAS_CDN = Object.freeze({
+  PRODUCTION: 'https://feaas.blob.core.windows.net',
+  STAGING: 'https://feaasstaging.blob.core.windows.net',
+});
+
+/**
+ * Walks through rendering tree and returns list of urls of all FEAAS Component Library Stylesheets that are used
+ * @param {LayoutServiceData} layoutData Layout service data
+ * @param {string} [hostname] CDN hostname, default is production hostname @see FEAAS_CDN urls
+ * @returns {string[]} library stylesheet urls
+ */
+export function getFEAASLibraryStylesheetURLs(layoutData: LayoutServiceData, hostname?: string) {
```

Comment:
Maybe rename the hostname parameter to serverUrl (since it's more than just hostname).
```suggestion
export function getFEAASLibraryStylesheetURLs(layoutData: LayoutServiceData, serverUrl?: string) {
```

Reply:
we call it `cdn` in sdk

Reply:
I decided to use `serverUrl` since we used this variable name in the sample